account-integrations/webauthn/src/lib/FCL_Webauthn.sol

```diff
@@ -47,55 +48,49 @@ library FCL_WebAuthn {
             bytes(challengeEncoded).length
         );
 
-            assembly {
-                calldatacopy(
-                    add(challengeExtracted, 32),
-                    add(clientData.offset, clientChallengeDataOffset),
-                    mload(challengeExtracted)
-                )
-            }
-
-            bytes32 moreData; //=keccak256(abi.encodePacked(challengeExtracted));
-            assembly {
-                moreData := keccak256(add(challengeExtracted, 32), mload(challengeExtracted))
-            }
-
-            if (keccak256(abi.encodePacked(bytes(challengeEncoded))) != moreData) {
-                revert InvalidClientData();
-            }
+            // TODO: wax#44 Extract webauthn challenge from clientData and compare to clientChallenge
```

Comment:
This logic didn't work out of the box, have added an issue to revisit this at a later stage